app/shared/db/project.server.ts

```diff
@@ -96,11 +105,19 @@ export const clone = async (clonableDomain: string): Promise<Project> => {
 
   const tree = await db.tree.clone(clonableProject.prodTreeId);
   const domain = generateDomain(clonableProject.title);
-
   const [project] = await Promise.all([
     prisma.project.create({
       data: {
-        userId: clonableProject.userId,
+        User: {
+          connectOrCreate: {
+            create: {
+              id: clonableProject.userId as string,
```

Comment:
same

Reply:
not part of this PR, but I just noticed the same thing below `parseProject(project as BaseProject) as Project;`
Instead we should treat the values as they are assumed by compiler instead of making compiler take it